### ISSUE TITLE
Costal chondritis

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11379]
**Preferred term label:**

**Synonyms**
Costochondritis

**Definition (free text, please give PubMed ID)**
PMID: 32914210 Our patient also had features suggesting RP: respiratory tract chondritis (hoarseness, shortness of breath), cochlear chondritis (vertigo and tinnitus), and 99mTc-MDP bone scintigraphy suggested **costal chondritis**.
Moreover, these two clinical entities responded to GC dose increasing and received remission with immunosuppressive agents. Thus, we were able to confirm the diagnosis of MAGIC syndrome in the present case.

**Parent term (use hpo.jax.org/app)**
Chondritis HP:0100662

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
MAGIC syndrome (mouth and genital ulcer with inflamed cartilage) ORPHA:324972

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001447
- label: inflammation of the costal cartilage, the cartilage that connects the ribs to the sternum (breastbone).
- parents: [Abnormal rib morphology (HP:0000772)](https://hpo.jax.org/app/browse/term/HP:0000772) | [Chondritis (HP:0100662)](https://hpo.jax.org/app/browse/term/HP:0100662)
- pmid: PMID:34264599
- def: inflammation of the costal cartilage (the cartilage that connects the ribs to the sternum). The typical presentation of costochondritis is bilateral parasternal chest wall pain exacerbated by deep breaths, coughing, and stretching. The upper (predominantly second through fifth) costochondral and/or costosternal junctions are most commonly involved. The areas of tenderness are not generally accompanied by heat, erythema, or localized swelling.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11379
- synonyms:
